# This file is distributed under the same license as the Django package.
#
# Translators:
# Ali Nikneshan <ali@nikneshan.com>, 2012-2013,2020
# Alireza Savand <alireza.savand@gmail.com>, 2013
# Ali Vakilzade <ali.vakilzade@gmail.com>, 2015
# Jannis Leidel <jannis@leidel.info>, 2011
# Pouya Abbassi, 2016
# Saeed <sd.javadi@gmail.com>, 2011
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-01-16 14:29+0330\n"
"PO-Revision-Date: 2023-12-04 18:45+0000\n"
"Last-Translator: Ali Nikneshan <ali@nikneshan.com>, 2012-2013,2020\n"
"Language-Team: Persian (http://app.transifex.com/django/django/language/"
"fa/)\n"
"Language: fa\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: env/lib/python3.10/site-packages/django/contrib/gis/apps.py:9
msgid "GIS"
msgstr "جی‌آی‌اس"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:81
msgid "The base GIS field."
msgstr "پایه‌ی سیستم اطلاعات جغرافیایی میدانی."

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:233
msgid ""
"The base Geometry field — maps to the OpenGIS Specification Geometry type."
msgstr "پایه‌ی فیلد هندسی --نوع خاص هندسه‌ی نقشه‌ها‌ی نوع OpenGIS"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:332
msgid "Point"
msgstr "نقطه"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:339
msgid "Line string"
msgstr "رشته خط"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:346
msgid "Polygon"
msgstr "چندضلعی"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:353
msgid "Multi-point"
msgstr "چند نقطه ای"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:360
msgid "Multi-line string"
msgstr "چند خط رشته"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:367
msgid "Multi polygon"
msgstr "چندین چند ضلعی "

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:374
msgid "Geometry collection"
msgstr "مجموعه هندسی"

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:380
msgid "Extent Aggregate Field"
msgstr "تراکم وسعت حوزه‌ی "

#: env/lib/python3.10/site-packages/django/contrib/gis/db/models/fields.py:395
msgid "Raster Field"
msgstr "حوزه‌ی شطرنجی"

#: env/lib/python3.10/site-packages/django/contrib/gis/forms/fields.py:21
msgid "No geometry value provided."
msgstr "مقدار جغرافیایی‌ای مقرر نشده است."

#: env/lib/python3.10/site-packages/django/contrib/gis/forms/fields.py:22
msgid "Invalid geometry value."
msgstr "مقدار جغرافیایی نامعتبر"

#: env/lib/python3.10/site-packages/django/contrib/gis/forms/fields.py:23
msgid "Invalid geometry type."
msgstr "نوعِ جغرافیایی نامعتبر"

#: env/lib/python3.10/site-packages/django/contrib/gis/forms/fields.py:25
msgid ""
"An error occurred when transforming the geometry to the SRID of the geometry "
"form field."
msgstr "مشکلی در هنگام انتقال مختصات هندسی از فیلد به SRID رخ داد."

#: env/lib/python3.10/site-packages/django/contrib/gis/templates/gis/openlayers.html:6
msgid "Delete all Features"
msgstr "حذف همه‌ی اشیاء"

#: env/lib/python3.10/site-packages/django/contrib/gis/templates/gis/openlayers.html:7
msgid "Debugging window (serialized value)"
msgstr "پنجره‌ی اشکال زدایی (ارزش متوالی)"

#: env/lib/python3.10/site-packages/django/contrib/gis/views.py:8
msgid "No feeds are registered."
msgstr "هیچ فیدی ثبت شده است."

#: env/lib/python3.10/site-packages/django/contrib/gis/views.py:14
#, python-format
msgid "Slug %r isn’t registered."
msgstr "Slug %r ثبت نشده"
